deploy/helm/sumologic/values.yaml

```diff
@@ -909,8 +908,10 @@ fluentd:
     ## Override output section for events. Leave empty for the default output section
     overrideOutputConf: |-
 
+    ## Source name for the Events source. Default: "events"
+    sourceName: "events"
     ## Source category for the Events source. Default: "{clusterName}/events"
-    sourceCategory: ""
+    # sourceCategory: "clustername/events"
```

Comment:
`"clustername/events"` can be misleading as this is not the actual default

Reply:
What would you prefer, `<cluster_name>/events`?

Reply:
I think `kubernetes/events`